web/application.py

```diff
@@ -16,13 +16,10 @@
 from . import webapi as web
 from . import wsgi
 from .debugerror import debugerror
-from .py3helpers import PY2, is_iter, iteritems, string_types
+from .py3helpers import is_iter, iteritems
```

Comment:
Let's drop `iteritems` and use `list(dict.items())` instead.

Reply:
This PR already has many changes, let's drop `py3helpers` in next PR.